docker/docker-compose.yml

```diff
@@ -34,7 +32,10 @@ services:
 
   # For the stationlite psql database
   psql:
+    container_name: stationlite-psql
     image: postgres:11
+    env_file:
+      - .env
```

Comment:
`.env` seems to be already the default value (see: https://docs.docker.com/compose/environment-variables/)

Reply:
Better to be explicit than implicit? I didn't know this and would probably be confused reading the docker-compose and seeing missing environment variables.

Reply:
Fine for me. I thought it is worth a note :+1:.